src/utils/etherscanAPI.ts

```diff
@@ -0,0 +1,4 @@
+
+
+
+export {}
```

Comment:
We are setting this Etherscan API in another PR, you might want to delete this to avoid any merge conflicts later